docs/src/pages/landing-page/Index.vue

```diff
@@ -168,21 +210,87 @@
       <q-img class="absolute-bottom" width="50%" src="~assets/landing-page/homepage-background-images/astronaut-left-hand.png" />
       <q-img width="50%" class="absolute-top-right" src="~assets/landing-page/homepage-background-images/astronaut-right-hand.png" />
     </div>
+
+    <div class="text-center lg-mb-large">
+      <q-icon size="xl" name="img:homepage-icons/medal.svg" />
+      <div class="lp-heading lp-heading-large">Our Sponsors</div>
+      <div class="lp-heading lp-heading-small">Every space odyssey has its patrons</div>
+      <div class="row justify-center">
+        <div class="col-8">
+          <div class="q-my-md">Platinum Sponsors</div>
+          <q-img :src="src" width="200px" v-for="(src, i) in sponsorLogos.platinum" :key="i" />
+          <div class="q-my-md">Gold Sponsors</div>
+          <q-img :src="src" width="200px" v-for="(src, i) in sponsorLogos.gold" :key="i" />
+          <div class="q-my-md">Silver Sponsors</div>
+          <q-img :src="src" width="200px" v-for="(src, i) in sponsorLogos.silver" :key="i" />
+        </div>
+      </div>
+    </div>
+
+    <div class="text-center final-words-section">
```

Comment:
I'd change `final-words-section` with something like `social-channels-call-to-action` or similar, describe the semantic meaning, not what it is into the layout (unless you're writing a reusable component which act as a layout manager, which is not the case)